addons/sale/models/product_template.py

```diff
@@ -29,8 +29,7 @@ class ProductTemplate(models.Model):
          ('sales_price', 'Sales price')
         ], string='Re-Invoice Expenses', default='no',
         compute='_compute_expense_policy', store=True, readonly=False,
-        help="Expenses and vendor bills can be re-invoiced to a customer."
-             "With this option, a validated expense can be re-invoice to a customer at its cost or sales price.")
+        help="Validated expenses and vendor bills can be re-invoiced to a customer at its cost or sales price.")
```

Comment:
Why is this change done in stable ?
Why does this PR target 16.3 when the task has the tag "Master" ?
Why isn't the PR in draft if the task is still in `RD DEV` ?

Reply:
I've made the pr in stable since it's mostly display improvement since milk has been released, so i know that a -u might be needed but i think it can be relevant to do it in 16.3. Now i can cherry pick the commit in a master pr if you think that it doesn't give any value to the 16.3.

I was waiting to do the last testing before putting it in green in rd-testing

Reply:
Well, at least for this label, if there is no specific reason to do it in stable, master should be the target.
Also, considering the size of your diff in the .pot, either the translations were not exported correctly in 16.3, or you didn't export it the same way they do the export, or a lot of bugfixes landed without updating the pot as they should have.  Anyway, it's strange, and another good reason to do it in master instead.

Reply:
I will do the label and pot in another PR then thanks for the review :smile: